server/src/test/java/org/elasticsearch/index/analysis/AnalysisTests.java

```diff
@@ -27,9 +31,18 @@
 import java.util.Arrays;
 import java.util.List;
 
+import static org.elasticsearch.index.analysis.AnalyzerComponents.createComponents;
 import static org.hamcrest.Matchers.is;
 
 public class AnalysisTests extends ESTestCase {
+    private static TestAnalysis testAnalysis;
+    private static Settings settings = Settings.builder().put(IndexMetadata.SETTING_VERSION_CREATED, Version.CURRENT).build();
+
+    @BeforeClass
+    public static void setup() throws IOException {
+        testAnalysis = createTestAnalysis(new Index("test", "_na_"), settings);
+    }
```

Comment:
do you need this as before class? It looks like it's only used in the new method that you introduced?